rpg-docs/Model/Character/Buffs.js

```diff
@@ -23,7 +23,7 @@ Schemas.Buff = new SimpleSchema({
 	type: {
 		type: String,
 		allowedValues: [
-			"inate",
+			"inate", //this should be "innate", but changing it could be problematic
 			"custom",
 		],
 	},
```

Comment:
I'm unsure whether we even need the "type" attribute here - any previous "innate" buffs (like Encumbered) are now Conditions, and all Buffs should be from CustomBuffs.

Reply:
will remove this, typo aside, it's not needed anymore